internal/server/data_source.go

```diff
@@ -91,6 +91,33 @@ func getCommonDataSchema() map[string]*schema.Schema {
 			Computed: true,
 			Elem:     &schema.Schema{Type: schema.TypeInt},
 		},
+		"network": {
+			Type:     schema.TypeSet,
+			Optional: true,
+			Elem: &schema.Resource{
+				Schema: map[string]*schema.Schema{
+					"network_id": {
+						Type:     schema.TypeInt,
+						Required: true,
```

Comment:
This field should be `Computed: true` instead of required, as the user can not specify it but the provider sets it.